portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,14 +43,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        
         String comment = request.getParameter("comment-box");
+        long timestamp = System.currentTimeMillis();
+        Entity commentEntity = new Entity("Comment");
```

Comment:
It'd be helpful to either comment what you're doing here:

```
// Persists comment as an entity in datastore.
...
```

Or extract this code into a helper function named `persistCommentInDatastore` and call it from this function.

This makes it clearer to the reader what is going on.

Reply:
done